responsemanager/responseassembler/responseassembler.go

```diff
@@ -150,6 +163,9 @@ func (rs *responseStream) execute(operations []responseOperation) {
 		size += op.size()
 	}
 	rs.messageSenders.AllocateAndBuildMessage(rs.p, size, func(builder *messagequeue.Builder) {
+		_, span = otel.Tracer("graphsync").Start(ctx, "message-build", trace.WithLinks(trace.LinkFromContext(builder.Context())))
```

Comment:
this introduces a new naming convention, maybe this should be `messageBuild`, or do you see this as a special case? if so, how would you describe that case, so we can be consistent with it?

Reply:
no you're write I'm being careless